cmake/MirCommon.cmake

```
@@ -70,7 +70,7 @@ function (mir_discover_tests_internal EXECUTABLE TEST_ENV_OPTIONS DETECT_FD_LEAK
 
   if(ENABLE_MEMCHECK_OPTION)
     set(test_cmd ${VALGRIND_CMD} ${test_cmd_no_memcheck})
-    set(test_no_memcheck_filter "*DeathTest.*:ClientLatency.*")
+    set(test_no_memcheck_filter "*DeathTest.*" "ClientLatency.*")
```

Slightly more explicit maybe?

```suggestion
    list(APPEND test_no_memcheck_filter "*DeathTest.*" "ClientLatency.*")
```